cmd/stork/stork.go

```diff
@@ -238,6 +238,25 @@ func run(c *cli.Context) {
 	} else if err != nil {
 		log.Warnf("Unable to create stork version configmap: %v", err)
 	}
+	// create configmap to store stork admin namespace details when
+	// it differs from default admin namespace
+	adminNs := c.String("admin-namespace")
+	if adminNs != defaultAdminNamespace {
+		storkControllerCm := &api_v1.ConfigMap{}
+		storkControllerCm.Name = k8sutils.StorkControllerConfigMapName
+		storkControllerCm.Namespace = defaultAdminNamespace
+		storkControllerCm.Data = make(map[string]string)
+		storkControllerCm.Data[k8sutils.AdminNsKey] = c.String("admin-namespace")
+		_, err = schedops.Instance().CreateConfigMap(storkControllerCm)
```

Comment:
Why is the cm getting created only if adminNs != defaultAdminNamespace?

Reply:
Otherwise, it will be kube-system namespace by default. So we might be needing it.

Reply:
Yes, we wont be creating configmap if we don't have a custom admin namespace. As kube-system is the default one

Reply:
I was checking on the later modification of admin namespace, so when user removes the custom admin namespace. The config map was still holding older one. So modified the code , to create configmap even there is no custom admin namespace. 

By default, the configmap now will hold `kube-system` when admin namespace is not set